Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM centos/ruby-25-centos7
 
+ENV RAILS_ENV=development
+ENV RACK_ENV=development
+ENV SECRET_KEY_BASE=cannot-be-blank-for-production-env-when-building
+ENV RAILS_SERVE_STATIC_FILES=1
```

Comment:
maybe we should remove this here and add it to docker-compose as variable to make standalone run of this container possible.